src/routes/profile.js

```diff
@@ -155,47 +198,176 @@ router.get(
   },
 );
 
+const storage = multer.memoryStorage();
+const upload = function (req, res, next) {
+  multer({
+    storage: storage,
+    limits: { fileSize: MAX_FILESIZE },
+  }).fields([
+    { name: 'avatarRes', maxCount: 1 },
+    { name: 'avatarSpy', maxCount: 1 }, // Whitelist, other files will not be accepted
+  ])(req, res, function (err) {
+    // TODO-kev: Check below if it is handling it correctly
+    if (err instanceof multer.MulterError && err.message === 'File too large') {
```

Comment:
Let's pull this error msg check inside this if statement. If it is file too large, then tack your 1mb limit string. 

Else, send back the error as is provided by multer.